README.md

```diff
@@ -36,6 +36,44 @@ Notice BR supports building with Go version `Go >= 1.13`
 
 When BR is built successfully, you can find binary in the `bin` directory.
 
+## Get started in 2 minutes!
```

Comment:
unless you can prove that these can complete in 2 minutes I'd rather just say "Quick start" here. (The `go-ycsb` step generating N=100,000 via INSERT will probably take a long time)